### ISSUE TITLE
Enable sidestep by default since it can no longer be abused

### ISSUE THREAD

**Issue by tommy61157** [PiratesAhoy/new-horizons#61]
Before, Sidestep was disabled in internalsettings.h (Towards the bottom of the file) since people used to be able to clip through walls with it before, however, people can no longer do that with the newest engine, so I would propose to enable sidestep by default since it can no longer be abused and can only be used in combat like it should now.

**Comment by Hammie** [comment]
I do feel like we need new animations before enabling this? Always felt a bit clunky to me.

**Comment by tommy61157** [comment]
Yeah, there still is no proper animation for it, maybe that can be part of the implementation I suppose. Would be willing to test if we could get some animations created.
